akka-http-core/src/main/scala/akka/http/impl/engine/ws/WebSocketClientBlueprint.scala

```diff
@@ -53,7 +53,7 @@ object WebSocketClientBlueprint {
     val valve = StreamUtils.OneTimeValve()
 
     val (initialRequest, key) = Handshake.Client.buildRequest(uri, extraHeaders, subprotocol.toList, settings.websocketRandomFactory())
-    val hostHeader = Host(uri.authority)
+    val hostHeader = Host(uri.authority.normalizedFor(uri.scheme))
```

Comment:
I'll take another look at that in a little while. Thanks for raising the concern.